app/gui/src/dashboard/hooks/projectHooks.ts

```diff
@@ -331,40 +344,42 @@ export function useOpenProject() {
 
   const enableMultitabs = useFeatureFlag('enableMultitabs')
 
-  return eventCallbacks.useEventCallback((project: LaunchedProject) => {
-    if (!canOpenProjects) {
-      return
-    }
-
-    if (!enableMultitabs) {
-      // Since multiple tabs cannot be opened at the same time, the opened projects need to be closed first.
-      if (projectsStore.getState().launchedProjects.length > 0) {
-        closeAllProjects()
+  return eventCallbacks.useEventCallback(
+    (project: LaunchedProject & { cloudProjectId?: backendModule.ProjectId }) => {
```

Comment:
What's the purpose of keeping  `cloudProjectId` property separately from the `LaunchedProject` prop?

Reply:
This is an extra parameter I need to pass when starting the project in the Hybrid mode. We need an original id of the cloud project to upload it after it is closed.

Reply:
I mean, why we won't add it to `LaunchedProject` interface, or introduce another interface that extends the `LaunchedProject` ?

Reply:
Ok, I updated the schema